packages/client/hmi-client/src/page/data-explorer/components/tera-search-results-list.vue

```diff
@@ -2,12 +2,12 @@
 	<div class="result-details">
 		<span class="result-count">
 			<template v-if="isLoading">Loading...</template>
-			<template v-else-if="props.searchTerm">
+			<template v-else-if="props.searchTerm || searchByExampleOptionsStr">
 				{{ resultsText }}
 				<span v-if="searchByExampleOptionsStr.length === 0"> "{{ props.searchTerm }}" </span>
 				<div v-else-if="searchByExampleOptionsStr.length > 0" class="search-by-example-card">
 					<tera-asset-card
-						:asset="searchByExampleAssetCardProp"
+						:asset="searchByExampleItem as Document & Model & Dataset"
```

Comment:
Shouldn't it be `Document | Model | Dataset`?

Reply:
You're right, it should. For whatever reason the expected type for the `asset` prop was `Document & Model & Dataset` instead of `ResultType` (which is an alias for `Document | Model | Dataset`. I've gone ahead an updated the expected `asset` prop type as well as its usage everywhere else in code.